src/calculateRentalCost.js

```diff
@@ -5,6 +5,18 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  let totalCost = cost * days;
+
+  if (days >= 7) {
+    return (totalCost -= 50);
```

Comment:
Replace the hard-coded value '7' with a descriptive constant like 'LONG_TERM_RENTAL_DAYS' and '50' with 'LONG_TERM_DISCOUNT'. Also, avoid using the compound assignment operator '-=' when returning a value. It's clearer to return 'totalCost - LONG_TERM_DISCOUNT'.